### DISCUSSION TITLE
Aktualisierungsfortschritt - Firmware

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#2213] [🇩🇪 Fragen&Antworten] [answered]
Ist es möglich, Raspberrymatic dazu zu bringen, den Update-Fortschritt anzuzeigen?
Es geht nur um die Aktualisierung der Firmware von HmIP-Geräten

**Comment by jens-maus** [comment, marked as answer]
Nein, geht nicht weil all die Geräte-Firmware Updateroutinen im closed source teil (HMIPServer) der CCU/OCCU Software liegt und folglich nur von eQ3 angepasst werden kann. Tut aber auch nicht weh. Einfach hochladen und warten auch wenn es Stunden, Tage oder manchmal auch Wochen dauern kann...
